plugins/woocommerce-blocks/assets/js/editor-components/ces-feedback-button/index.tsx

```diff
@@ -0,0 +1,123 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	CustomerEffortScoreModalContainer,
+	useCustomerEffortScoreModal,
+} from '@woocommerce/customer-effort-score';
+import { Button, TextareaControl, TextControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import FeedbackIcon from './feedback-icon';
+
+interface CesFeedbackButtonProps {
+	blockName: string;
+	title?: string;
+	firstQuestion?: string;
+	feedbackLabel?: string;
+	feedbackPlaceholder?: string;
+	emailLabel?: string;
+	emailHelp?: string;
+	buttonText?: string;
+	submitLabel?: string;
+}
+
+export const CesFeedbackButton = ( {
+	blockName,
+	title = __( 'Share your experience', 'woocommerce' ),
+	// translators: %s is the block name.
+	firstQuestion = __(
+		'It was easy for me to accomplish what I wanted with the %s.',
+		'woocommerce'
+	).replace( '%s', blockName ),
+	// translators: %s is the block name.
+	feedbackLabel = __(
+		`How can we improve the %s block for you? (Optional)`,
+		'woocommerce'
+	).replace( '%s', blockName ),
```

Comment:
I think we should use `sprintf` function from `@wordpress/i18n`

```suggestion
	firstQuestion = sprintf( __(
		'It was easy for me to accomplish what I wanted with the %s.',
		'woocommerce'
	), blockName ),
	// translators: %s is the block name.
	feedbackLabel = sprintf( __
		`How can we improve the %s block for you? (Optional)`,
		'woocommerce'
	), blockName ),
```